internal/daysteps/daysteps.go

```diff
@@ -11,10 +17,66 @@ const (
 	mInKm = 1000
 )
 
+var (
+	ErrInvalidArgumentCount = errors.New("expected 2 arguments")
+	ErrZeroSteps            = errors.New("steps must be greater than zero")
+	ErrZeroDuration         = errors.New("duration must be greater than zero")
+)
+
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	dataSlice := strings.Split(data, ",")
+
+	if len(dataSlice) != 2 {
+		return 0, 0, ErrInvalidArgumentCount
+	}
+
+	steps, err := strconv.Atoi(dataSlice[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше всегда оборачивать ошибку
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```